common/tasks/weighted_channel.go

```diff
@@ -25,7 +25,7 @@
 package tasks
 
 const (
-	WeightedChannelDefaultSize = 100
+	WeightedChannelDefaultSize = 10000
```

Comment:
This can be regarded as the host level task buffer. and handle a burst of task submission. The size should be at least bigger than the priority weight. Set it to a larger number so that when there's a burst of tasks, instead of waiting for a periodical reschedule (which happens 3-5s later), buffer the tasks in the scheduler can shorten the e2e latency since they will be processed as soon as possible.